ext/standard/tests/array/array_map_variation16.phpt

```diff
@@ -36,21 +28,11 @@ for($count = 0; $count < count($callback_names); $count++)
 echo "Done";
 ?>
 --EXPECT--
-*** Testing array_map() : non-permmited built-in functions ***
--- Iteration 1 --
```

Comment:
I dislike it too, but there are so many tests structured like this, I'm not sure it makes sense to remove the headers just here.

Reply:
The main reason is so that I didn't need to do the renumbering of the later iterations